src/components/node-list/styles/node-list.scss

```diff
@@ -9,10 +9,81 @@
 @import './row';
 @import './filter';
 
+.pipeline-nodelist {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  padding-top: 1.8em;
+
+  &:before,
+  > * {
```

Comment:
The cover element doesn't sit on top, it sits underneath, and changes the background colour. I'm using it (instead of just changing the background colour) in order to be able to just toggle the opacity value (between 0 to 1) rather than toggle the background colour which is less performant.

Reply:
I'd suggest simplifying as you can get a close enough effect I think with roughly:

```css
&--fade:before {
  ...
  z-index: 1;
  background: #eaecef;
  opacity: 0.7;
}
```

Maybe I missing something here?

Reply:
Yeah, because the intention is not only to reduce the opacity of the text, it's also to change the colour of the background. A blue overlay at 70% opacity on top of a green background won't look blue, it'll be teal. I want the background colour to be 100% opaque. Illustration here, with changed colours for clarity:

How it works currently:
<img width="300" src="https://user-images.githubusercontent.com/1155816/97294341-c319f380-1845-11eb-981c-041503b6cfe8.png"> <img width="300" src="https://user-images.githubusercontent.com/1155816/97294370-cdd48880-1845-11eb-9f8d-fac38107d646.png">

How it would look if I changed the overlay to 70% opacity:
<img width="300" src="https://user-images.githubusercontent.com/1155816/97294562-0e340680-1846-11eb-8473-0eca891c3d5f.png">

Do you see how the background colour bleeds through? I am trying to make the background 100% the colour I am after, and not a 70/30 mix. The overlay background colour is different.

Reply:
Yeah I thought this was what you meant, but I was thinking it's such a subtle difference given the additional complexity, but no problem then if you'd prefer to keep it for the effect.

Reply:
It's done now, it's consistent with the design as it was when I did it, which I think made more sense as a design (as it much better achieved the effect of making the dropdown stand out from the rest of the sidebar). So I think it makes sense to leave it as it. Thanks!